swap_meet/vendor.py

```diff
@@ -1,2 +1,93 @@
-class Vendor:
-    pass
+class Vendor():
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category_string):
+        list_by_category =[]
+        for item in self.inventory:
+            if item.category == category_string:
+                list_by_category.append(item)
+        return list_by_category
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item in self.inventory and their_item in vendor.inventory:
+            self.inventory.remove(my_item)
+            vendor.inventory.append(my_item)
+            vendor.inventory.remove(their_item)
+            self.inventory.append(their_item)
+            return True
+        else:
+            return False
+
+    def swap_first_item(self, vendor):
```

Comment:
You already have a function that will do this work for you! swap_items() can be called here and then you would pass in the first item of the vender and the other vendor as arguments
return self.swap_items(other, self.inventory[0], other.inventory[0])